packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -454,7 +449,7 @@ export async function getVenueCollections(
   const openCollections = collections.items.filter((c) => c.open);
 
   if (openCollections.length === 0) {
-    session.log.info(`${chalk.red(`🚦 venue "${venue}" is not accepting submissions.`)}`);
+    session.log.info(`${chalk.red(`🚦 venue "${venue}" has no open collections.`)}`);
```

Comment:
made this more specific, the "not accepting submissions." not only happens in 1 place and as a result of the `v1/sites/{name}/access` call.